src/components/Header/components/Nav.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Nav.scss';
+import NAV_LIST from '../navData';
+const Nav = ({ isActive, showNav }) => {
+  return (
+    <nav className={isActive ? 'nav active' : 'nav'}>
+      <div className="dim" onClick={showNav} />
+      <div className="nav_box">
+        <section className="menu_top">
+          <h3 className="menu_name">메뉴</h3>
+          <ul className="icon_menu">
+            <li className="cart">
+              <i className="fa-solid fa-cart-shopping" />
+            </li>
+            <li className="close" onClick={showNav}>
+              <i className="fa-solid fa-xmark" />
+            </li>
+          </ul>
+        </section>
+        <section className="menu_list">
+          <h4 className="menu_name">쇼핑하기</h4>
+          <ul className="catogry_list">
+            {NAV_LIST.map(data => (
+              <li key={data.id}>
+                <img src={data.image} alt={data.catogory} />
+                <p className="category_name">{data.catogory}</p>
+              </li>
+            ))}
```

Comment:
key prop까지 잘 부여해주셨네요 👍
return되는 부분이 그렇게 복잡한 부분은 아니지만, 구조분해할당하여 사용하시면 조금 더 복잡한 구조에서 보다 더 깔끔하게 작성하실 수 있겠네요.
```suggestion
            {NAV_LIST.map(({ id, image, category }) => (
              <li key={id}>
                <img src={image} alt={catogory} />
                <p className="category_name">{catogory}</p>
              </li>
            ))}
```